Changelog.md

```diff
@@ -4,6 +4,9 @@ Expect active development and potentially significant breaking changes in the `0
 
 ### vNext
 
+### 1.0.0-rc.4
```

Comment:
You can just put this under vNext, and we'll figure out which version we'll release it under.